Casamari/MNC007.xml

```diff
@@ -36,76 +39,168 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                              >136–139</citedRange></bibl>).</note>
+                     <incipit xml:lang="gez">
+                        <locus target="#21v"/>
+                        <hi rend="rubric">ኦእግዚኦ፡ መሐረነ፡ ኦእግዚኦ፡ መሐከነ፡ ኦእ</hi>ግዚኦ፡ መሐርከነ፡ ኦሥሉስ፡ ቅዱስ፡
+                        አምላክነ፡ ወተስፋነ፡ መሐር፡ ልህኵትከ፡ አድኅን፡ ነፍሳተነ፡ </incipit>
+                     <explicit xml:lang="gez">
+                        <locus target="#44v"/>ዘአምትረትከ፡ ሄሮድያዳ፡ በግፍዕ፡ እንበለ፡ እዳ፡ ሰ፡ ዕበ፡ ስእንኩ፡
+                           ጸ<supplied reason="lost" resp="SH">ዲቀ</supplied>፡ ለኦሪተ፡ ሙሴ፡ <supplied
+                           reason="lost" resp="SH">በ</supplied>ሕጋ፡ <gap reason="lost" unit="chars"
+                           quantity="7"/> እንበለ፡ ንትጋ፡ <supplied reason="lost" resp="SH"
+                        >ዘአ</supplied>ርኃውከ፡ ቍልፈ፡ መ<add place="interlinear">ን</add>ሰጋ፡ ሰ፡ <gap
+                           reason="illegible" unit="chars" quantity="4"/> ጻማየ፡ እግዚኦ፡ ከመ፡ <gap
+                           reason="illegible" unit="chars" quantity="5"/> በከንቱ፡ ዮሐንስ፡ ሀበኒ፡
+                           ዘ<supplied reason="lost" resp="SH">እ</supplied>ፈቱ፡ እስመ፡ <gap
+                           reason="illegible" unit="lines" quantity="3"/> ማርያም፡ ወለጸሐፊሁ፡ ወ<supplied
+                           reason="lost" resp="SH">ል</supplied>ደ፡ ቂር<gap reason="illegible"
+                           unit="chars" quantity="2"/></explicit>
+                     <note>Most probably the name of <persName ref="PRS12029GabraMa">Gabra
+                           Māryām</persName> is stated here at the very end together with that of a scribe,
+                      probably named Walda Qirqos (<ref target="#h3"/>).</note>
                   </msItem>
-                  <msItem xml:id="ms_i3">
-                     <locus from="21v" to="32r"/>
-                     <title type="complete">Malkǝʾa Māryām</title>
+
+                  <msItem xml:id="ms_i4" corresp="#h2">
+                     <locus from="23r" to="32r"/>
+                     <title type="complete" ref="LIT2891RepCh172"><foreign xml:lang="gez">Malkǝʾa
+                           Māryām</foreign></title>
                      <textLang mainLang="gez"/>
-                     <note>
-                     </note>                
+                     <note>The work comprises 60 stanzas.</note>
+                     <incipit xml:lang="gez">
+                        <locus target="#23r"/>
+                        <pb n="23r"/>
+                        <hi rend="rubric">ሰላም፡ ሰላም፡</hi> ለዝክረ፡ ስምኪ፡ ሐዋዝ፡ እምነ፡ ከልበኔ፡ ወቍስጥ፡ ወእምነ፡
+                        ሰንበልት፡ ምዑዝ፡ <hi rend="rubric">ማርያም፡</hi> ለባሲተ፡ ዓቢይ፡ ትእዛዝ፡ ይስቅየኒ፡ ለለጽባሁ፡ ወይነ፡
+                        ፍቅርኪ፡ አዚዝ፡ ከመ፡ ይሰቅዮ፡ ውሂዝ፡ ዘሰናይ፡ አርዝ ።</incipit>
+                     <explicit xml:lang="gez">
+                        <locus target="#32r"/>
+                        <hi rend="rubric">ስብሐት፡</hi> ለኪ፡ <hi rend="rubric">ማርያም፡</hi> በኍልቈ፡ ኵሉ፡
```

Comment:
The correct form is the one you wrote, but the ms. has ኁ instead of ኍ. You can propose also here the ኍ as a correction. Since the form ኁ instead of ኍ is consistent in the ms. (this is not an isolated case) this can be maybe due to a phonetic interference of the dialect spoken by the scribe.

Reply:
Yes, you are right. However, since the handwriting is not clear, I intepreted it as first a ኍ. 
I, however, followed your comment and changed it in my record with my first update today. It is already marked up now :)